DIFD/models/difd_1year_moisture0_veg1_fence0.txt

```diff
@@ -0,0 +1,186 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% AeoLiS model configuration          default values within ()     %%
+%% Date: 2023-02-08                                                 %%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% -------------------- [Timing] ----------------------------- %%
+dt                      = 7200                            % (60) [s] Timeinterval between timesteps    
+tstart                  = 0                               % (0.)[s] Starting time of the simulation
+tstop                   = 24397200                        % (3600)[s] End time of the simulation
+restart                 = None                            % (None) [s] Interval for which to write restart files
+refdate                 = 2023-25-06                      % ('2020-01-01 00:00')[-] Reference datetime in netCDF output
+
+%% -------------------- [Grid files *.grd] ------------------- %%
+xgrid_file              = ../data/grids/x.grd             % (None) Filename of ASCII file with x-coordinates of grid cells
+ygrid_file              = ../data/grids/y.grd             % (None) Filename of ASCII file with y-coordinates of grid cells
+bed_file                = ../data/grids/z.grd             % (None) Filename of ASCII file with bed level heights of grid cells
+ne_file                 = ../data/grids/ne.grd            % (None) Filename of ASCII file with non-erodible layer
+veg_file                = ../data/grids/veg.grd           % (None) Filename of ASCII file with initial vegetation density
+threshold_file          = None                            % (None) Filename of ASCII file with shear velocity threshold
+fence_file              = None                            % (None) Filename of ASCII file with sand fence location/height (above the bed)
+wave_mask               = None                            % (None) Filename of ASCII file with mask for wave height
+tide_mask               = None                            % (None) Filename of ASCII file with mask for tidal elevation
+threshold_mask          = None                            % (None) Filename of ASCII file with mask for the shear velocity threshold
+
+%% -------------------- [Time series *.txt] ------------------ %%
+tide_file               = ../data/bc/tide.txt             % (None) Filename of ASCII file with time series of water levels
+wave_file               = ../data/bc/waves.txt            % (None) Filename of ASCII file with time series of wave heights (and periods)
+wind_file               = ../data/bc/wind.txt             % (None) Filename of ASCII file with time series of wind velocity and direction
```

Comment:
which wind file is used? I had two: one transformed to beach conditions and one from regional station